### ISSUE TITLE
faster width/ height input

### ISSUE THREAD

**Issue by illustree** [ANGIstudio/Sketch-Resize-Commands#4]
Hi there,
what about adding the following:

just inputting “100,50” would set to width=100 and height=50 ...  instead of having to type in “w=100,h=50” which takes too much time ... many thanks in advance

**Comment by andrejilderda** [comment]
Hi @illustree, I'm working on a new version of Resize commands and I'll definitely look into your suggestion. Thanks for your comment!

**Comment by ErikFontanel** [comment]
Just chiming in as someone who uses this plugin a lot but did not had any time yet to fork it… A faster syntax would much appreciated.

Are you aware of the [keyboard resizing](https://mastersketchup.com/drawing-accurately-sketchup/) in SketchUp (the 3D software) @ajilderda ? I have a lot to dislike about SketchUp, but the one thing they got right is the resizing. It'd be great if someday, somehow that always visible, always ready dimensions input would be available in Sketch too. But let's start with the syntax.

**Comment by andrejilderda** [comment]
Hi @ErikFontanel, nice to hear this plugin is useful to you! About the dimensions, I assume you refer to this screenshot? ![Sketchup Dimensions](https://mastersketchup.com/wp-content/uploads/2015/03/dimensions.jpg) 
Do you have suggestions how this could be applied to this plugin (knowing that some characters `, / + - % = #` are reserved for math expressions)?

Good to know if you're thinking of forking this project, I've restarted working on the entire rewrite of this plugin here with more options and a better GUI:
<img width="632" alt="screen shot 2017-11-23 at 12 19 55" src="https://user-images.githubusercontent.com/487182/33170471-a19e5f04-d048-11e7-8062-11bd682ed00e.png">
PR's and suggestions are more than welcome:
https://github.com/ajilderda/Sketch-Commander

**Comment by ErikFontanel** [comment]
Yeah kind, but those dimensions matter because of the different units it supports.

This explains it well:
![](https://discourse-cdn.global.ssl.fastly.net/sketchup/uploads/default/original/4/4/44f90ba81d33dcc648281282bce2d397a3e87c16.gif)

It works as follows:
* Start drawing a shape in SketchUp
* Before finishing your shape, lift your hand off the mouse
* Type in the exact dimensions you need, you don't have to focus or call an input
* press Enter and your shape is drawn with the dimensions specified.

[Video](https://www.youtube.com/watch?v=k_H4MxF7e-Q)

This measurements box supports also math operators and relative lengths. Of course I realise Sketch works a bit different internally, but the gist of the SketchUp system is really nice.

I like your idea of upgrading the interface. It would be even nicer if [Sketch Runner](http://sketchrunner.com/) would offer some input API to send inputs to plugin actions…
